antbs/database/package.py

```diff
@@ -28,12 +28,12 @@
 import time
 import zipfile
 
-from gevent import sleep
-from github3 import login
 from gitlab import Gitlab
```

Comment:
Imports can be sorted.

*PyImportSortBear, severity NORMAL, section `python`.*

The issue can be fixed by applying the following patch:

```diff
--- a/antbs/database/package.py
+++ b/antbs/database/package.py
@@ -28,12 +28,11 @@
 import time
 import zipfile
 
-from gitlab import Gitlab
-
 from database.base_objects import RedisHash
 from database.server_status import status
 from gevent import sleep
 from github3 import login
+from gitlab import Gitlab
 from utils.logging_config import logger
 
 REPO_DIR = "/var/tmp/antergos-packages"
```